services/galley/src/Galley/Cassandra/Conversation.hs

```diff
@@ -56,6 +56,66 @@ import Wire.API.MLS.CipherSuite
 import Wire.API.MLS.Group
 import Wire.API.MLS.PublicGroupState
 
+createMLSSelfConversation ::
```

Comment:
Does this function really need to take in a whole `NewConversation` object? I don't think we should be handling team conversations and proteus conversations here, since this function is specific to MLS self conversations.

Reply:
Fair points! Addressed in commit e2de44d76.